doc/governance/governance.rst

```diff
@@ -0,0 +1,124 @@
+   :header-rows: 1
+
+   * - Name
+     - Affiliation
+     - GitHub handle
+     - PMC role
+     - Time Zone (optional)
+   * - Bence Maygar
+     - `Locus Robotics <https://locusrobotics.com/>`_
+     - `bmagyar <https://github.com/bmagyar>`_
+     - Project Leader
+     - GMT (UTC+0)
+   * - Denis Stogl
+     - `b»robotized <https://www.b-robotized.com/>`_
+     - `destogl <https://github.com/destogl>`_
+     - Project Co-Leader
+     - CET (UTC+1)/CEST (UTC+2)
+   * - Christoph Fröhlich
+     - `AIT - Austrian Institute of Technology GmbH <https://www.ait.ac.at/>`_
+     - `christophfroehlich <https://github.com/christophfroehlich>`_
+     - Member
+     - CET (UTC+1)/CEST (UTC+2)
+   * -  Sai Kishor Kothakota
+     - `PAL Robotics S.L <https://pal-robotics.com/>`_
+     - `saikishor <https://github.com/saikishor>`_
+     - Member
+     - CET (UTC+1)/CEST (UTC+2)
+
+Current ros-controls Committers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ros-controls committers (who are not also part of the ros-controls PMC) consists of the following constituents:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Affiliation
+     - GitHub handle
+     - Time Zone (optional)
+   * - Alejandro Hernandez Cordero
+     - `Honu Robotics <https://www.honurobotics.com/>`_
+     - `ahcorde <https://github.com/ahcorde>`_
+     - CET (UTC+1)/CEST (UTC+2)
+
+Past ros-controls PMC Constituents
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ros-controls PMC thanks the following past constituents for their service:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - PMC role
+     - GitHub handle (optional)
+   * - None yet
+     - None yet
+     - None yet
+
+Repositories managed by the ros-controls PMC
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following repositories are managed by the ros-controls PMC:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Repository URL
+     - Committers
+   * - https://github.com/ros-controls/ros2_control
+     - Not Yet Available
+   * - https://github.com/ros-controls/ros2_controllers
+     - Not Yet Available
+   * - https://github.com/ros-controls/ros2_control_cmake
+     - Not Yet Available
+   * - https://github.com/ros-controls/ros2_control_ci
+     - Not Yet Available
+   * - https://github.com/ros-controls/ros2_control_demos
+     - Not Yet Available
+   * - https://github.com/ros-controls/control_msgs
+     - Not Yet Available
+   * - https://github.com/ros-controls/control_toolbox
+     - Not Yet Available
+   * - https://github.com/ros-controls/control.ros.org
+     - Not Yet Available
+   * - https://github.com/ros-controls/gz_ros2_control
+     - Alejandro Hernandez Cordero
+   * - https://github.com/ros-controls/kinematics_interface
+     - Not Yet Available
+   * - https://github.com/ros-controls/realtime_tools
+     - Not Yet Available
+   * - https://github.com/ros-controls/ros2_rhel
+     - Not Yet Available
+   * - https://github.com/ros-controls/ros2_debian
+     - Not Yet Available
```

Comment:
Can you explain the reason for Not yet Available?

Reply:
this means that there are no committers yet for this repository

Reply:
copied from here https://docs.ros.org/en/rolling/The-ROS2-Project/Governance.html#repositories-managed-by-the-ros-pmc

Reply:
Got it. Thanks for the confirmation